### PR TITLE
[RATOM-78] handles html

### PR DESCRIPTION
**Pull request by michael-caktus**
remove <script> and <style> tags from html emails, adds bs4 to requirements

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ djangorestframework-simplejwt==4.4.0
 django-elasticsearch-dsl==7.1.0
 django-elasticsearch-dsl-drf==0.20.5
 
+bs4==0.0.1
```
Comment (JeremyGibson) [inline requirements/base.txt]: this seems weird? BeautifulSoup is on version 4.8.2.
https://pypi.org/project/beautifulsoup4/

**Review by JeremyGibson — CHANGES_REQUESTED** [review]
Check into the version number on BeautifulSoup. Other than that looks good.

File: requirements/base.txt (hunk 1 above)
Comment (michael-caktus) [reply requirements/base.txt]: I had it at 4.8.2, but it broke-- said there was no bs4 4.8.2. I think whatever abstraction of BeautifulSoup bs4 is, is on version 0.0.1.  Are you able to include bs4 using 4.8.2?

Comment (michael-caktus) [reply requirements/base.txt]: Switched it!

**Pull request merged**